reset-dev-env.bash

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+npm install
+
+poetry install --extras='cdk datasets-endpoint'
+
+# shellcheck source=/dev/null
+. .venv/bin/activate
```

Comment:
Why not using `pyenv activate` ?

Reply:
I didn't know that was a thing. I'll look into it.

Reply:
It looks like [the subproject](https://github.com/pyenv/pyenv-virtualenv/issues) is dead, and besides the command doesn't work:

```
$ pyenv activate 
pyenv-virtualenv: version `3.8.6' is not a virtualenv
```

Reply:
Thanks. I had the same message and was wondering if I am doing something wrong.